app/src/main/java/com/codesoom/assignment/application/ProductSaveRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.Product;
+
+import java.math.BigDecimal;
+
+/**
+ * 고객이 입력한 정보를 엔티티로 변환하는 역할을 담당합니다.
+ */
```

Comment:
조금 미묘한 설명 같습니다. 가끔은 쿨하고 단호하게 주석을 적어보는 연습을 하는 것도 괜찮겠죠.
```suggestion
/**
 * 상품 정보 저장 요청.
 */
```